Cloud.md

```diff
@@ -66,11 +57,190 @@ Relevant experience:
+
+- I have a wide understanding of cloud development technology and practices, backed by extensive practical experience.
+- I am confident making architectural decisions taking concerns like infrastructure, identity management, security, scalability, performance, concurrency and maintainability into consideration.
+- I can apply my technical abilities to productively solve business problems with confidence and pragmatism.
+- I use my skills, speed and knowledge to help my team be very productive.
+
+#### I am investing back into Loan Market.
+
+- I am becoming a trusted voice in Loan Market and my opinions and ideas seem to carry some weight.
+- I am mentoring another team member, deliberately helping them to round out their skill set.
+- I am sharing my on-the-job learning and experiences with others so they can be more effective in their roles.
+- I am involved in initiatives beyond cloud engineering.
+
+---
+
+## Lead Engineer
+> I am a well-rounded developer, architect, problem solver and leader of people. I am trusted to lead teams to deliver a small number of pitches from inception to their successful completion.
+>
+> 🔀 This role is the start of the Management track.
+
+#### I am becoming a respected and trusted leader of people.
+
+- I am emerging as a leader of people.
+- I can confidently step outside my comfort zone and adapt quickly to new team situations.
+- I am comfortable diving head-first into moderately risky environments with relatively little upfront information and oversight.
+- I am comfortable with transparently assessing risk, making recommendations, escalating appropriately and dealing with the consequences along the way.
+- I can confidently and charismatically pitch ideas and influence my teams decisions.
+- I am an adept communicator, and can effectively steer technical and non-technical conversations to positive outcomes over any medium.
+
+#### I am learning to be a servant leader for my team.
+
+- I am prioritising the needs of my team members above my own, knowing that their success is our success.
+- I am learning to reliably deliver successful projects by applying good practices when leading teams.
+- I am focussed on helping my peers grow, sharing what I have learned and encouraging others to do the same.
+- I facilitate ongoing, net-positive and productive communication in my team.
+- I participate and have a strong voice in the hiring of new team members.
+- I understand that I cannot solve every problem for my team by doing it myself. I am learning to trust and delegate tasks to my team.
+
+#### I am emerging as a driver for technical and business opportunities.
+
+- I am looking for new technologies and practices that maybe able to assist our team to achieve their goals and improve how we work.
+- I am comfortable with building a product vision with an Engineering Manager based on the business needs, regardless of whether this involves technology or otherwise.
```

Comment:
```suggestion
- I am comfortable with building a product vision with a Product or Engineering Manager based on the business needs, regardless of whether this involves technology or otherwise.
```